Get HZ value from system rather than requiring via CPPFLAGS

Patch by Robin Getz <rgetz@blackfin.uclinux.org>

--- a/dhry.h
+++ b/dhry.h
@@ -359,6 +359,10 @@
                 /* for "times" */
 #endif
 
+#ifndef HZ
+#include <sys/param.h>
+#endif
+
 #define Mic_secs_Per_Second     1000000.0
                 /* Berkeley UNIX C returns process times in seconds/HZ */
 
